wgpu-native/src/command/compute.rs

```diff
@@ -33,11 +33,11 @@ impl<B: hal::Backend> ComputePass<B> {
 
 #[no_mangle]
 pub extern "C" fn wgpu_compute_pass_end_pass(pass_id: ComputePassId) -> CommandBufferId {
-    let pass = HUB.compute_passes.unregister(pass_id);
```

Comment:
would it be feasible to have `compute_passes` (and other things inside `HUB`) to be strongly typed by the ID?

Reply:
That's what I tried to do first, but I have some hard time with it.
It would certainly be better.